octobot_backtesting/importers/exchanges/exchange_importer.py

```diff
@@ -77,27 +79,32 @@ def __init__(self, config, file_path):
             ohlcv_min_timestamps = (await self.database.select_min(ExchangeDataTables.OHLCV,
                                                                    [DataBase.TIMESTAMP_COLUMN],
                                                                    [CONFIG_TIME_FRAME],
-                                                                   group_by=CONFIG_TIME_FRAME))
+                                                                   group_by=CONFIG_TIME_FRAME,
+                                                                   **where_clauses_kwargs
+                                                                   ))
 
-            ohlcv_timestamp = max(ohlcv_min_timestamps)[0]
-            ohlcv_max_time_frame = max([TimeFramesMinutes[tf] for tf in self.time_frames])
-            min_ohlcv_timestamp = ohlcv_timestamp - ohlcv_max_time_frame
+            min_ohlcv_timestamp = max(ohlcv_min_timestamps)[0]
+            # ohlcv_max_time_frame = max([TimeFramesMinutes[tf] for tf in self.time_frames])
+            # min_ohlcv_timestamp = ohlcv_timestamp - ohlcv_max_time_frame
```

Comment:
@Herklos What was this for ? Now that we start exactly at the start of the 1st candle I think it's now useless

Reply:
Yes it's useless now :+1: